data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/actions/CountAggregateActionConfig.java

```diff
@@ -18,6 +19,9 @@ public class CountAggregateActionConfig {
     @JsonProperty("count_key")
     String countKey = DEFAULT_COUNT_KEY;
 
+    @JsonProperty("name")
```

Comment:
I'm wondering if we should use a different property name here. I see that this matches what OTel uses - `name`. But, at the same time it is a little unclear. Perhaps `metric_name`? I'm also ok keeping what you have here.